### ISSUE TITLE
Imports: Glitch during simultaneous imports

### ISSUE THREAD

**Issue by dmsnell** [Automattic/wp-calypso#2497]
There is a race condition that exists when multiple sessions are attempting to upload an export file for the same site at the same time. When this happens, the first upload to finish will start an import session, but the second one will receive an error that an import is already in progress. However, on that screen, the first import session that began will also pop in form the polling in the background, showing two WordPress importer cards where there should have only been a single one.

The chances of this happening are rare, as a second upload needs to start while a first upload is currently uploading. Outside of this time, the already-started import session should appear on the screen in Calypso when navigating to the import section, leaving no opportunity to start a new one.

The fix for this should probably be a check on the error from the API. If the error concerns simultaneous uploads, the new and temporary session should be deleted in favor of the one that was just started.

Spotted in #2486

**Comment by dmsnell** [comment]
D1064 merged in r131225-wpcom and deployed

**Comment by lancewillett** [comment]
In wp-admin I noticed this flow is a bit different:

1. Start an import from /wp-admin/admin.php?import=wordpress
2. Close the tab (or reload) after it kicks off.
3. You'll see a notification saying "Processing… We're processing your import now and will send you an email when it's all done. If you don't hear from us within 24 hours, please contact support, and we'll get everything fixed."

![screen shot 2016-02-19 at 13 21 33](https://cloud.githubusercontent.com/assets/66797/13187992/e63e7d70-d70b-11e5-9e14-67e5d5ff3b00.png)


In Calypso, the import doesn't show a status (at least not in my testing).

1. Start an import from /settings/import/example.wordpress.com
2. Close the tab, or reload.
3. Instead of a waiting "Pending" message, it looks like the UI is at the starting point, with "Start Import" button active and ready to go.

![screen shot 2016-02-19 at 13 22 25](https://cloud.githubusercontent.com/assets/66797/13187987/e2609a62-d70b-11e5-947a-3def7c7d8c08.png)

**Comment by lancewillett** [comment]
Note: I didn't test wp-admin start and Calypso status check (or vice versa) — that seems too edge case-y.

**Comment by hoverduck** [comment]
12May16 Bug Scrub

Confirmed that this is still an issue, I ended up with two Import cards on the session that finished the upload first.  However, I did not see an error on the other session saying that an import was already in progress.  As seen below, the new card that popped in gets updated status from the import in progress, but attempting to use the first import card just yields a spinner.  The session with the spinner generated this error in the JS console: `Uncaught (in promise) Error: Import not found.(…)`

![wp-calyps-2497](https://cloud.githubusercontent.com/assets/7233112/15218678/034016b8-182f-11e6-8e2f-c09ebecf186b.gif)
